.tmux.conf.local

```diff
@@ -39,6 +39,23 @@ tmux_conf_new_session_prompt=false
 
 # -- display -------------------------------------------------------------------
 
+# color palette
+# reference as "${color_name}"
+# N.B., double quotes are necessary for interpolation
+#       single quotes mean string literal
+blue='#0087d7'
+light_blue='#00afff'
+gray='#444444'
+black='#000000'
+yellow='#ffff00'
+light_gray='#8a8a8a'
+dark_gray='#080808'
+red='#d70000'
+green='#5fff00'
+orange='#ff5f00'
+pink='#ff00af'
+white='#e4e4e4'
+
```

Comment:
I added something similar to my `.tmux.conf.local`

```sh
# -- colors --------------------------------------------------------------------
# ASCII colors
ascii_black='black'
ascii_red='red'
ascii_green='green'
ascii_yellow='yellow'
ascii_blue='blue'
ascii_magenta='magenta'
ascii_cyan='cyan'
ascii_white='white'

# HEX colors
hex_black='#000000'
hex_red='#ff0000'
hex_green='#00ff00'
hex_blue='#0000ff'
hex_yellow='#ffff00'
hex_white='#ffffff'

hex_pink='#ff00af'

hex_light_blue='#00afff'
hex_mid_blue='#0087d7'

hex_light_gray='#8a8a8a'
hex_gray='#444444'
hex_dark_gray='#080808'

# Color mapping
black=$ascii_black
red=$ascii_red
green=$ascii_green
yellow=$ascii_yellow
blue=$ascii_blue
magenta=$ascii_magenta
cyan=$ascii_cyan
white=$ascii_white

pink=$hex_pink

light_blue=$hex_light_blue
mid_blue=$hex_mid_blue

light_gray=$hex_light_gray
gray=$hex_gray
dark_gray=$hex_dark_gray
```

The `# ASCII colors` can be changed by the color presets, like "Solarized Dark", of iTerm.